.github/workflows/node-ci.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       uses: Khan/actions@full-or-limited-v0
       with:
         full-trigger: ${{ steps.eslint-reset.outputs.filtered }}
-        full: yarn lint:ci
+        full: yarn lint:ci .
```

Comment:
Without the `.` eslint was only running on the .js files in the root directory.  I don't know why this is the default eslint behavior, it feels like such a footgun. 😞